web/src/app/WebApp/Installers/Flarum/FlarumSetup.php

```diff
@@ -0,0 +1,211 @@
+<?php
+namespace Hestia\WebApp\Installers\Flarum;
+
+use Hestia\System\Util;
+use Hestia\WebApp\Installers\BaseSetup as BaseSetup;
+
+class FlarumSetup extends BaseSetup {
+	protected $appInfo = [
+		"name" => "Flarum",
+		"group" => "forum",
+		"enabled" => true,
+		"version" => "latest",
+		"thumbnail" => "fl-thumb.png",
```

Comment:
Could we use one of our official branding images here rather than modifying an older version of the logo?

I've attached two options below.

![symbol name](https://user-images.githubusercontent.com/7406822/216846944-ee20834d-9c27-4ebf-86b6-2f45151d6334.svg)

![on gradient orange](https://user-images.githubusercontent.com/7406822/216846941-6b7ad2ca-ee1f-46e8-b873-ecb54f443d2b.svg)